.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Reproducibility**
+- [ ] I can reproduce the bug consistently.
+- [ ] The bug reproduces only sometimes.
+- [ ] I was only able to observe the bug once.
+
+**To Reproduce**
+Steps to reproduce the behavior. For example:
+1. When using Brevitas layer-wise/PTQ/export feature '...'
+2. With model '....'
+3. And then running '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots/Code Example/Log Output**
+If applicable, add screenshots, code blocks, or output text to help explain your problem.
+
+**please complete the following information:**
+ - Version number of Brevitas:
```

Comment:
Add also a line for pytorch version and OS system

Reply:
Perhaps also give the option for unknown configuration so if they observed an issue but no longer can recall the setup, the default isn't to give default versions/os

Reply:
I would leave the suggestion to include pytorch and OS, they can always remove the lines and not follow the template.

Reply:
added